src/adapter/src/catalog/migrate.rs

```diff
@@ -324,6 +330,15 @@ fn ast_rewrite_rewrite_type_schemas_0_81_0(stmt: &mut Statement<Raw>) {
     Rewriter.visit_statement_mut(stmt);
 }
 
+fn persist_default_cluster_0_81_0(
+    txn: &mut Transaction<'_>,
+    _conn_catalog: &ConnCatalog<'_>,
+    _connection_context: &ConnectionContext,
+) -> Result<(), anyhow::Error> {
+    txn.upsert_system_config("cluster", "default".to_string())?;
```

Comment:
Do we need to check if they've already overwritten their `cluster` var and not `upsert` in that case?

Reply:
https://github.com/MaterializeInc/materialize/pull/23903 has yet to be included in a release, so it wouldn't have been previously possible for users to overwrite their `cluster` var in this case